gammapy/scripts/image_pipe.py

```diff
@@ -16,6 +20,16 @@
 class ImageAnalysis(object):
     """Gammapy 2D image based analysis.
 
+    Fill a `~gammapy.image.SkyMapCollection`:
```

Comment:
Yes, this is what I wanted.

You have to put bullets and an empty line before the first bullet for Sphinx to render it correctly.

Like this:
```
Fill a `~gammapy.image.SkyMapCollection`:

* counts : counts for one obs
* bkg : bkg model for one obs
```